tests/conftest.py

```diff
@@ -41,6 +45,7 @@ def pytest_configure(config: pytest.Config) -> None:
     os.environ.clear()  # lets always start with a clean environment to make tests consistent
     os.environ["ENVIRONMENT"] = "local"
     os.environ["APP_DATABASE_SCHEME"] = "sqlite"
+    os.environ["APP_DATABASE_FILE"] = "/database_test.sqlite3"
 
     config.uvicorn = {  # type: ignore
```

Comment:
propably better to rewrite this if you want to support 2 urls

Reply:
It would be nice if the tests could use a different database. I now have the tests empty my 'demo' database for regular running.